clap_derive/tests/subcommands.rs

```diff
@@ -161,6 +161,96 @@ fn test_tuple_commands() {
     assert!(!output.contains("Not shown"));
 }
 
+#[test]
+fn external_subcommand() {
+    #[derive(Debug, PartialEq, Clap)]
+    struct Opt {
+        #[clap(subcommand)]
+        sub: Subcommands,
+    }
+
+    #[derive(Debug, PartialEq, Clap)]
+    enum Subcommands {
+        // Add,
+        // Remove,
+        #[clap(external_subcommand)]
+        Other(Vec<String>),
+    }
+
+    // assert_eq!(
+    //     Opt::parse_from(&["test", "add"]),
+    //     Opt {
+    //         sub: Subcommands::Add
+    //     }
+    // );
+
+    // assert_eq!(
+    //     Opt::parse_from(&["test", "remove"]),
+    //     Opt {
+    //         sub: Subcommands::Remove
+    //     }
+    // );
```

Comment:
Please uncomment these asserts. We want to allow normal subcommands *with* external subcommands.